teuthology/test/test_reimage_error_mark_machine_down.py

```diff
@@ -0,0 +1,75 @@
+from teuthology.dispatcher import supervisor
+from unittest.mock import patch
+import unittest
```

Comment:
I'm curious about the decision to use `unittest` over `py.test` here

Reply:
From what I see from other unit tests in teuthology, a majority of them use a mix of pytest and unittest. Especially, patch from unittest.mock. I just find unittest documentation easier to follow and it is easier to mock functions. I've tested the unittest with pytest command as well and it works.

Reply:
Changed to use pytest, only using patching from unittest.mock